### PR TITLE
Rewrite docs theme

### PR DESCRIPTION
**Pull request by invisiblehermit**
Theme Style Changed on Apil

Main nav, side nav, dropdown menu, container-header, search bar, extra pages, and font format

### CODE REVIEW

File: docs-theme/css/extra.css [hunk 1]
```diff
@@ -0,0 +1,361 @@
...
+}
+
+
+
+
+
+
+
+.navbar-support {
+	color: #2fa4e7;
```
Comment (cmaujean) [inline docs-theme/css/extra.css]: spacing

File: docs-theme/css/extra.css [hunk 2]
```diff
@@ -0,0 +1,361 @@
...
+    margin-right: 15px;
+    margin-bottom: 8px;
+    background-color: transparent;
+    background-image: none;
+    border: 0;
+    border-radius: 4px;
+}
+
+.navbar-support:hover,.navbar-support:focus{
+	border: 0;
```
Comment (cmaujean) [inline docs-theme/css/extra.css]: more off spacing

File: docs-theme/css/extra.css [hunk 3]
```diff
@@ -0,0 +1,361 @@
...
+}
+
+.navbar-support .icon-bar {
+	background-color: #fff;
```
Comment (cmaujean) [inline docs-theme/css/extra.css]: again

File: docs-theme/css/extra.css [hunk 4]
```diff
@@ -0,0 +1,361 @@
...
+    padding: 3px 20px;
+    clear: both;
+    font-weight: normal;
+    line-height: 1.428571429;
+    color: #AFAFAF;
+    white-space: nowrap;
+}
+
+.dropdown-menu>.active>a{
+	color: #fff;
```
Comment (cmaujean) [inline docs-theme/css/extra.css]: spacing

Comment (cmaujean) [inline docs-theme/css/extra.css]: css should be 2 spaces indented and consistent.

File: docs-theme/nav.html [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+                            {%- elif config.repo_name == 'Bitbucket' -%}
+                                <i class="fa fa-bitbucket"></i>
+                            {%- endif -%}
+                            {{ config.repo_name }}
+                        </a>
+                    </li>
+                {%- endif %}
+              {%- endblock %}
+
+              <!--  <li>
```
Comment (cmaujean) [inline docs-theme/nav.html]: if this isn't going to be used, it should be erased, not commented out

File: docs-theme/css/extra.css (hunk 4 above)
Comment (invisiblehermit) [inline docs-theme/css/extra.css]: between where...

**Comment by erikh** [comment]
Ok, two bugs @invisiblehermit:

http://localhost:8000/ will bring me to a "page not found" when running `mkdocs serve` on my local machine. This should return the intro page.

The `User Guide` drop-down does not appear to work.

**Comment by erikh** [comment]
Additionally, can you bring the syntax coloring back? It was really nice.

**Comment by invisiblehermit** [comment]
OK let me fix these

**Comment by erikh** [comment]
Ok, just checked it out; looks great! Really great.

Let's get this merged. Contact me on slack, I need you to squash the repo.

**Comment by erikh** [comment]
@cmaujean We need to get this into the repository; can you give it a final review please?

**Comment by erikh** [comment]
@cmaujean I'm going to merge this, can you do any additional necessary work (like manage the release notes in the header) since eric has to get back to what he was doing before?

If not I can do this work, nbd

**Pull request merged**